api/v1beta1/foundationdbcluster_types.go

```diff
@@ -1231,13 +1231,13 @@ type FoundationDBClusterAutomationOptions struct {
 	// +kubebuilder:default:=Zone
 	RemovalMode PodUpdateMode `json:"removalMode,omitempty"`
 
-	// WaitTimeBetweenRemovals defines how long to wait between the last removal and the next removal. This is only an
-	// upper limit if the process group and the according resources are deleted faster then the provided duration the
+	// WaitDurationBetweenRemovals defines how long to wait between the last removal and the next removal. This is only an
+	// upper limit if the process group and the according resources are deleted faster than the provided duration the
 	// operator will move on with the next removal. The idea is to prevent a race condition were the operator deletes
 	// a resource but the Kubernetes API is slower to trigger the actual deletion and we are running into a situation
 	// where the fault tolerance check still includes the already deleted processes.
 	// Defaults to 60s.
-	WaitTimeBetweenRemovals time.Duration `json:"waitTimeBetweenRemovals,omitempty"`
+	WaitDurationBetweenRemovals time.Duration `json:"waitDurationBetweenRemovals,omitempty"`
```

Comment:
I think it would be more in line with the Kubernetes idioms if we called it something like `waitDurationBetweenRemovalsNanoseconds`, similar to fields like `terminationGracePeriodSeconds` and `activeDeadlineSeconds` in the pod spec.

Reply:
The difference is that those referenced fields are `int64` and not a `time.Duration`. Currently we already have the `ignorePendingPodsDuration` that also takes a `time.Duration` as input which means that the user has to specify the time unit e.g. `30s` or `5m` in the request.